AsyncDisplayKit/Details/ASImageContainerProtocolCategories.m

```diff
@@ -0,0 +1,37 @@
+//
+//  ASImageContainerProtocolCategories.m
```

Comment:
Great file!! <3 simplicity.  Less loving the category naming convention, but I agree it is necessary.